src/lib/tecs/TECS.hpp

```diff
@@ -120,7 +123,7 @@ class TECS
 
 	void set_equivalent_airspeed_max(float airspeed) { _equivalent_airspeed_max = airspeed; }
 	void set_equivalent_airspeed_min(float airspeed) { _equivalent_airspeed_min = airspeed; }
-	void set_equivalent_airspeed_cruise(float airspeed) { _equivalent_airspeed_cruise = airspeed; }
+	void set_equivalent_airspeed_trim(float airspeed) { _equivalent_airspeed_trim = airspeed; }
```

Comment:
wondering now why not to set the throttle trim mappings (for each of these calibrated airspeeds) in a similar manner with setters instead of inputting them every time to the update method? these six values form a trim map, which should be more of a static thing -- the trim map is then still adjustable with density if it's available. but only requires inputting the current density in a feedback manner.

Reply:
@tstastny I removed the entire trim throttle mapping out of TECS. It really seems nicer to do it outside, as I don't see any benefit of doing it in TECS rather than a bunch of params that need to be passed.